region.go

```diff
@@ -44,115 +44,149 @@ func GetRegion(zone string) (region string, err error) {
 
 // Region describes respective IBM Cloud COS region, VPC region and Zones associated with a region in Power VS.
 type Region struct {
-	Description string
-	VPCRegion   string
-	COSRegion   string
-	Zones       []string
-	SysTypes    []string
+	Description  string
+	VPCRegion    string
+	COSRegion    string
+	Zones        []string
+	SysTypes     []string
+	IPISupported bool
 }
 
 // Regions provides a mapping between Power VS and IBM Cloud VPC and IBM COS regions.
-var Regions = map[string]Region{
+var Regions = FullRegions
```

Comment:
Why do we need to do like this? Any reason behind this?
IMHO, we can use the existing map itself with the addition of IPISupported field on each region.